zib_uploader/zib_uploader.py

```diff
@@ -1,9 +1,78 @@
 # -*- coding: utf-8 -*-"""Documentation about the ZIB uploader module."""
 import logging
+from typing import Iterator, Tuple
+
+import pandas as pd
+
+from zib_uploader.fetch_data import OntologyMapping
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_MAPPING_PREDICATE = 'http://www.w3.org/2004/02/skos/core#closeMatch'
+DEFAULT_VALID_SOURCES = ('CUI',)
+
+
+class OntologyMappingsProcessor:
```

Comment:
Can you rename the module to `ontology_mappings_processor`? The `zib_uploader` module is another remnant of the escience project template.

Reply:
I have renamed it to `process_data.py`, so it is a bit more general.

Reply:
I have added a script in `scripts/get_ontology_mappings`. Can you check to see if you like it?